src/reducers/identity.js

```diff
@@ -107,6 +114,21 @@ const identity = (state = defaultState, action) => {
       });
     case SET_SHOW_EMPTY_CLAIM_CATEGORIES:
       return state.setIn(['personalInformation', 'showEmptyClaimCategories'], action.payload.value);
+    case SET_SHOW_HIDDEN_CLAIMS:
+      return state.setIn(['personalInformation', 'showHiddenClaims'], action.payload.value);
+    case SET_CLAIM_VISIBILITY:
+      return state.setIn(['personalInformation', 'claims', 'byId', action.payload.claim, 'hidden'], action.payload.value);
+    case UPDATE_SELECTED_CLAIMS:
+      return state.updateIn(['personalInformation', 'selectedClaims'], IList(), (selectedClaims) => {
+        if (selectedClaims.indexOf(action.payload.claim) > -1) {
```

Comment:
Why do you keep using indexOf?
Use include as stated before already. It's more radable.